src/compiler/utilities.ts

```diff
@@ -2197,9 +2197,6 @@ namespace ts {
     }
 
     function isVariableDeclarationInitializedWithRequireHelper(node: Node, allowAccessedRequire: boolean) {
-        if (node.kind === SyntaxKind.BindingElement) {
-            node = node.parent.parent;
-        }

```

Comment:
This is the problem code; other changes are accounting for the deletion. The issue is that this function gets consumed exclusively by functions whose return type is a predicate of `node` being some sort of `VariableDeclaration`, so this defeats that type predicate.